wazuh/wazuh-kibana-app
# Add Apple Silicon architecture to the register Agent wizard

### Description
Hi team,

this pull request adds a new button with the Apple Silicon architecture to the **Register Agents** wizard.

 
### Issues Resolved
Closes #5202 

### Evidence
![image](https://github.com/wazuh/wazuh-kibana-app/assets/9343732/1786d7d6-e03a-4944-8e05-c1d2ea746d2c)

### Test
To test the PR we need to check both macOS architecture work properly:

- Go to Agents
- Press the `Deploy new agent` button
- Press the macOS in option 1
- Select MacOS Sierra in the 2nd step
- Select each of the available architectures and verify the installation script is correct, the package name should be like `wazuh-agent-${VERSION}-${REVISION}.${ARCH}.pkg:`

<details>
<summary>Intel script</summary>

![Screenshot from 2023-06-19 09-29-50](https://github.com/wazuh/wazuh-kibana-app/assets/9343732/29e19ee1-106e-42cc-b4a4-cefc513f807f)

</details>

<details>
<summary>Apple Silicon script</summary>

![Screenshot from 2023-06-19 09-30-01](https://github.com/wazuh/wazuh-kibana-app/assets/9343732/17ae6a11-f6dd-401f-b523-11a2f8d3a8ee)

</details>

### Check List
